### DISCUSSION TITLE
A special tag for property-manipulation in other classes

### DISCUSSION THREAD

**Discussion by Timbuktu1982** [EnterTheNameHere/esdoc-monorepo#10] [Q&A]
Hey there,

give it a tag that indicates, that a own property was injected or manipulated (default value) from another class?

**Comment by EnterTheNameHere** [comment]
Can you write some example how it should look?

**Comment by Timbuktu1982** [comment]
Of course, with pleasure.

I work a lot with emberjs. This framework has a runloop. This runloop defines processes for the first start of apps or route changes.

Before "services" are generally available or templates are rendered, many things are done, e.g. process initializations. 
For better understanding: boot -> .... -> initializer -> ...... -> services.

In my case I access a service object in the initializer and inject things into this service object (e.g. settings, variables or other). And now I want to document this in the target service....

My idea for tag names are @preparedFrom (e.g. in case the variable are declared with default in the target object, but filled from another object) or  @injectedFrom (e.g. in case the variable are not existent in target object and declared in target object by another object).

**Comment by EnterTheNameHere** [comment]
I'll have a look at emberjs to get more context.
